src/Search_Replace_Command.php

```diff
@@ -649,6 +649,30 @@ private static function esc_sql_ident( $idents ) {
 		return array_map( $backtick, $idents );
 	}
 
+	/**
+	 * Puts MySQL string values in single quotes, to avoid them being interpreted as column names.
+	 *
+	 * @param string|array $values A single value or an array of values.
+	 * @return string|array A quoted string if given a string, or an array of quoted strings if given an array of strings.
+	 */
+	private static function esc_sql_value( $values ) {
+		$quote = function ( $v ) {
+			// Don't quote numeric values to MySQL's implicit type conversion.
+			if ( is_numeric( $v ) ) {
+				return esc_sql( $v );
+			}
+
+			// Put any string values between single quotes.
+			return "'" . str_replace( "'", "''", esc_sql( $v ) ) . "'";
```

Comment:
The `str_replace()` should not be there.

Reply:
Removed.